src/Microsoft.ML.Dnn/DnnRetrainTransform.cs

```diff
@@ -903,7 +907,7 @@ protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, b
 
                             var editor = VBufferEditor.Create(ref dst, (int)tensorSize);
 
-                            DnnUtils.FetchData<T>(tensor.ToArray<T>(), editor.Values);
+                            tensor.ToSpan<T>(editor.Values);
```

Comment:
I would name this method `CopyTo`

```C#
tensor.CopyTo(editor.Values);
```

Note that you can drop the generic on `CopyTo<T>` since it can be inferred from the parameter being passed in.